### ISSUE TITLE
NameError by using library

### ISSUE THREAD

**Issue by EdwardChest** [andreabak/sibilant#1]
```python
from sibilant.voip import VoIPPhone
voip = VoIPPhone(SIP_Username, SIP_Password, SIP_IP, SIP_Port)
voip.call('4444')
```

Python 3.10.9 [venv]
`
NameError: More than one Header subclass with the same _name "DEFAULT" defined: UnknownHeader and UnknownHeader
`

**Comment by andreabak** [comment]
Hello @DevCorg, thank you for the heads-up.

As you can probably guess, this project is still in very early stages.

I will look into the issue in the coming days, when time allows.

Feel free to suggest a fix / PR in the mean time.

**Comment by EdwardChest** [comment]
> Hello @DevCorg, thank you for the heads-up.
> 
> As you can probably guess, this project is still in very early stages.
> 
> I will look into the issue in the coming days, when time allows.
> 
> Feel free to suggest a fix / PR in the mean time.

I think this error caused by python version. Which version you use?

**Comment by andreabak** [comment]
@DevCorg should be now fixed with 156f4b66
